.github/workflows/release.yml

```diff
@@ -28,14 +29,17 @@ jobs:
         run: |
           export DYNAWO_HOME=$(pwd)
           util/envDynawo.sh build-user
+          util/envDynawo.sh build-doc
           util/envDynawo.sh distrib
           util/envDynawo.sh distrib-headers
           util/envDynawo.sh distrib-omc
           version=$(util/envDynawo.sh version | cut -f1 -d' ')
-          curl -H "authorization: Bearer $GITHUB_TOKEN" -H "Content-Type: application/zip" -X POST $(curl -s -H "authorization: Bearer $GITHUB_TOKEN" --request GET https://api.github.com/repos/dynawo/dynawo/releases/latest | grep upload_url | cut -d '"' -f 4 | grep -o ".*assets")?name=Dynawo_Linux_v${version}.zip --data-binary @distributions/Dynawo_omc_V${version}.zip
```

Comment:
Please do not modiify the curl line for zip

Reply:
I put **$(curl -s -H "authorization: Bearer $GITHUB_TOKEN" --request GET https://api.github.com/repos/dynawo/dynawo/releases/latest | grep upload_url | cut -d '"' -f 4 | grep -o ".*assets")** and **https://api.github.com/repos/dynawo/dynawo** in variables to improve the readability of the code